src/python/grpcio/grpc/_adapter/_types.py

```diff
@@ -321,6 +354,28 @@ def create_call(self, completion_queue, method, host, deadline=float('+inf')):
     """
     return None
 
+  @abc.abstractmethod
+  def connectivity_state(self, try_to_connect=False):
```

Comment:
When I look at the C API I see two separate methods (grpc_channel_watch_connectivity_state and grpc_channel_check_connectivity_state) - am I misreading the API?